### DISCUSSION TITLE
here is my allsky online ...

### DISCUSSION THREAD

**Discussion by TonAmSi** [aaronwmorris/indi-allsky#1178] [Show and tell]
https://allsky.astrodon.space/indi-allsky/ 

I also like to post about my allsky in Fediverse :-) 
-> astronomy.social/@2ndStar 

CS!

**Comment by aaronwmorris** [comment]
I apologize for missing this.  That is an awesome build!

Have you tried enabling stretching?

**Reply by TonAmSi** [reply]
Thank you!
Yes, every now and then.
But I've rarely had clear skies for months.
It's hardly worth it.

**Comment by NC48401** [comment]
Great site. This is what I would like to do but having a hard time finding tutorials. I am Pi and Linux alliterate and not a programmer. I was able to get TJ's allsky up and running but was able to find helpful articles. A point in the right direction would be appreciated.

**Reply by NeilAllison147** [reply]
I'm in the same boat as you :(
I have indi-allsky running with an ASI676MC camera but am now struggling to get any further (e.g tweaking settings, exposing to the outside world etc, etc).

**Reply by danteffm** [reply]
Hi @NeilAllison147 - I'm using the AS678MC and was tweaking day and night views over the last weeks. 

In my blog, I documented what I did, maybe that helps? 
https://allsky-rodgau.com/optimal-configuration-for-indi-allsky-stable-day-night-transition-and-reliable-moon-mode-1541/ 

![ccd6_20251225_035514](https://github.com/user-attachments/assets/aa34e6b3-09e3-4eee-8205-94ccaf74a0fb)

**Comment by OceanStateAstro** [comment]
How did you go about being able to access the local site from anywhere? I tried the Wiki but couldn't get anywhere.

**Reply by aaronwmorris** [reply]
There are a couple of different ways.  If you have the ability to setup a port forward on your router, you can just directly expose indi-allsky to the Internet.

The second way is by using the SyncAPI.  This method requires you to have a virtual private server [VPS] hosting a web-only installation of indi-allsky.  Your local indi-allsky would use a web services API to sync images and videos to the remote VPS.

https://github.com/aaronwmorris/indi-allsky/wiki/SyncAPI-Setup
